base_classes/NXhistory.nxdl.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2014-2024 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXhistory" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <doc>
+         A set of activities that occurred to a physical entity prior/during experiment.
+         
+         Ideally, a full report of the previous operations (or links to a chain of operations).
+         Alternatively, notes allow for additional descriptors in any format.
+    </doc>
+    <group type="NXactivity">
+        <doc>
+             Any activity that was performed on the physical entity prior or during the experiment.
+        </doc>
+    </group>
+    <attribute name="identifierNAME">
+        <doc>
+             An ID or reference to the location or a unique (globally
+             persistent) identifier of e.g. another file which gives
+             as many as possible details of the history event.
```

Comment:
I'm a little confused: does NXhistory represent a sequence of activities (a sequence of history events) or a single activity (a single history event).

The documentation mentions "the history event", suggesting NXhistory is describing a single history event, which (I think) isn't the intention.